### PR TITLE
[2] Building a website using HTML & CSS for the tech-career advisory organisation

### PR DESCRIPTION
**Pull request by ghost**
- Created the folders called home, inside it include images folder and home html file, path html and register html file
-Created a CSS folder which i  have include the CSS file
-Worked on the home page 
-Also added links to the other page like path and register page
-Below are images of  the full website
home_page
home-page
page-home

### CODE REVIEW

**Comment by ChabuChabu** [comment]
> * Created the folders called home, inside it include images folder and home html file, path html and register html file
>   -Created a CSS folder which i  have include the CSS file
>   -Worked on the home page
>   -Also added links to the other page like path and register page
>   -Below are images of  the full website
> 
> home-first
> home-last
> home-second
> home-third

File: Home/home.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <div class="nav-header">
+            <h1><a href="#top">Zamcademy</a></h1>
+        </div>
+        <div class="nav-content">
+          <ul>
+            <li><a href="#bottom">About Us</a></li>
```
Comment (ChabuChabu) [inline Home/home.html]: Kindly remove the about us href

Comment (ghost) [reply Home/home.html]: its done

**Comment by ChabuChabu** [comment]
> * Created the folders called home, inside it include images folder and home html file, path html and register html file
>   -Created a CSS folder which i  have include the CSS file
>   -Worked on the home page
>   -Also added links to the other page like path and register page
>   -Below are images of  the full website
> 
> home-first
> home-last
> home-second
> home-third

Remove all screenshots and leave only the one we need

**Review by ChabuChabu — APPROVED** [review]

**Review by Luuba — APPROVED** [review]

**Pull request merged**